### ISSUE TITLE
TF Plan crashes when aws_wafregional_web_acl contains a rule group

### ISSUE THREAD

**Issue by erikpaasonen** [terraform-providers/terraform-provider-aws#4077]
Edit:  added `-auto-approve` in the Steps to Reproduce.

### Terraform Version
Terraform v0.11.5
- provider.aws v1.13.0

### Affected Resource(s)
- aws_wafregional_web_acl

### Terraform Configuration Files
```hcl
resource "aws_wafregional_rule" "example" {
  name        = "example"
  metric_name = "example"
}

resource "aws_wafregional_rule_group" "example" {
  name        = "example_group"
  metric_name = "example"

  activated_rule {
    action {
      type = "COUNT"
    }

    priority = 50
    rule_id  = "${aws_wafregional_rule.example.id}"
  }
}

resource "aws_wafregional_web_acl" "waf_acl" {
  name        = "tfWebACL"
  metric_name = "tfWebACL"

  default_action {
    type = "ALLOW"
  }

  rule {
    action {
      type = "BLOCK"
    }

    priority = 1
    rule_id  = "${aws_wafregional_rule_group.example.id}"
  }
}
```

### Debug Output
https://gist.github.com/erikpaasonen/2269f54f7d4317d41e38825bf6267c60

### Panic Output
https://gist.github.com/erikpaasonen/e29967981728bf61ce1a9f8a956fd709

### Expected Behavior
Terraform should cleanly delete the WebACL.

### Actual Behavior
Observed Terraform crash, and the WebACL still exists in the AWS Console. It also still has the rule group attached.

### Steps to Reproduce
1. `terraform apply -auto-approve` with the above code
1. Due to a separate bug, the rule group won't actually attach to the WebACL, so the above Apply step will fail. So go to the AWS Console and attach the rule group named "example_group" to the WebACL.
1. `terraform destroy -auto-approve`

### Important Factoids
Support for the WAF Regional feature set was recently added to the AWS provider. This is likely just an oversight during initial implementation.

### References
- #4052 
- #3898 
- #424

**Comment by erikpaasonen** [comment]
Found that even a `terraform plan` crashes in this scenario, so this issue is not necessarily specific to the 'destroy' command.

**Comment by bflad** [comment]
This should be fixed with #5053 which was just merged into master and will release with version 1.27.0 of the AWS provider, likely middle of this week. 👍

**Comment by bflad** [comment]
This has been released in [version 1.27.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1270-july-11-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
